packages/vue/src/internal-components/IResizePane/IResizePane.vue

```diff
@@ -0,0 +1,168 @@
+<script setup lang="ts">
+import { computed, onMounted, ref, shallowRef, watchEffect } from "vue";
+import { debounce } from "@fkui/logic";
+import { useEventListener } from "../../composables";
+import { useAreaData } from "../../components/FPageLayout";
+import { useKeyboardHandler } from "./use-keyboard-handler";
+import { type Orientation } from "./orientation";
+import { usePointerHandler } from "./use-pointer-handler";
+import { type SizeState } from "./size-state";
+import { clamp } from "./clamp";
+import { computeCssValue } from "./compute-css-value";
+import { aggregateCssValue } from "./aggregate-css-value";
+import { useStorage } from "./use-storage";
+
+const STEP_SIZE = 10;
+
+const props = withDefaults(
```

Comment:
Borde funka med destruct? Men kanske inte medför någon fördel.
```suggestion
const { disabled, min, max, initial } = withDefaults(
```

Reply:
Tidigare krockade det med `min`, `max` och `initial` computed men nu heter de `minSize`, `maxSize` och `initialSize` så kan ändra tillbaka till destruct.

Reply:
@dvitamin gjorde precis ändringen men märkte att dokumentationen inte plockar upp default-värden längre då :(